server-common/src/main/java/org/apache/kafka/server/common/MetadataVersion.java

```diff
@@ -208,17 +208,20 @@ public enum MetadataVersion {
     // Support ListOffsetRequest v9 for KIP-1005.
     IBP_3_9_IV0(21, "3.9", "IV0", false),
 
+    // Bootstrap metadata version for version 1 of the GroupVersion feature (KIP-848).
+    IBP_4_0_IV0(22, "4.0", "IV0", false),
```

Comment:
I reordered the MVs. `IBP_4_0_IV0` is only used for bootstrapping the `group.version` feature flag. The ELR moved to `IBP_4_0_IV1`.